src/pages/incomplete.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from "react"
+import { Box, Button, Typography } from "@material-ui/core"
+import { navigate } from "gatsby"
+import { useDispatch, useSelector } from "react-redux"
+import { selectEntries, upsertEntry } from "../features/entries"
+import { selectMainMensesTag } from "../features/settings"
+import LastDateInput from "../features/onboarding/Onboarding/LastDateInput"
+import { AppLayout, AppMainToolbar, AppPage } from "../features/app"
+import Toast from "../features/app/Toast"
+
+const Incomplete = () => {
+  const dispatch = useDispatch()
+  const [lastPeriod, setLastPeriod] = useState(new Date())
+  const [error, setError] = useState(false)
+  const mainMensesTag = useSelector(selectMainMensesTag)
+  const entries = useSelector(selectEntries)
+  const notHasPlacedPeriod =
+    !entries.length || !entries.filter((s) => s.tags.length).length
+
+  useEffect(() => {
+    if (!notHasPlacedPeriod) {
+      navigate("/timeline")
+    }
+  })
+
+  const calculatePeriodDate = async () => {
+    setError(false)
+
+    try {
+      await dispatch(
+        upsertEntry(lastPeriod, {
+          note: `#${mainMensesTag}`,
+        })
+      )
+      navigate("/timeline")
+    } catch (e) {
+      setError(true)
+    }
+  }
+  return (
+    <AppLayout>
+      <AppMainToolbar>
+        <Typography variant="h6" component="h1">
+          Just one more step
+        </Typography>
+      </AppMainToolbar>
+      <AppPage withPaper>
+        <p>
+          Seems you have not yet told us when your last period was and
+          unfortunately we need that to calculate the next one.
+        </p>
+        <p>No rush, please tell us whenever you can.</p>
+        <Box mt={4}>
+          <LastDateInput value={lastPeriod} onChange={setLastPeriod} />
```

Comment:
**nitpick:** Styling of LastDateInput
```
inputProps={{
  color: "secondary",
  variant: "outlined",
  margin: "normal",
}}
```

Reply:
**though (non-blocking):** Could just be made to be default, without the need to be able to override as it should be consistent. Just me guilty of over generalization.

Reply:
Updated and changed the label aswell

![image](https://user-images.githubusercontent.com/1051509/118111367-df494080-b3e3-11eb-9863-a916dc47da32.png)

Reply:
It still does not look the same as the other one tho, any idea why?

Reply:
Yes you need to replace 

```
{...inputProps}
inputVariant={inputProps.variant}
```

straight on the `Datepicker` component, instead of on the `InputProps` and `InputLabelProps` 😅

Reply:
OHHHH! We got this!

Will do this today so you can merge tomorrow morning

Reply:
Did you forget to push? If not no problem, I am off Monday (Constitution day) FYI.

Reply:
I had to write this:

```
InputLabelProps={{
          color: "secondary",
          className: classes.label,
        }}
        inputVariant="outlined"
        InputProps={{
          color: "secondary",
          margin: "normal",
```

Seems it works well now and it's the default